licenses.d2iq.yaml

```diff
@@ -13,6 +13,7 @@ ignore:
   #       ref: 82a186f7a71ca66269dba0a3eef1fb16f9121946
   #       license_path: LICENSE
   - docker.io/nginxinc/nginx-unprivileged:1.22.0-alpine
+  - docker.io/library/nginx:1.22
```

Comment:
You can probably remove this one as we switch to the final image

Reply:
the license scan failed because of this earlier https://github.com/mesosphere/kommander-applications/actions/runs/4066750062
should I still remove it?

Reply:
Yeah, the image should be gone now because of #945

Reply:
[859698d](https://github.com/mesosphere/kommander-applications/pull/937/commits/859698df023c1271c82d927dfb09e9aab0113291)